src/types/device/deviceApi.ts

```diff
@@ -44,4 +44,16 @@ export default {
     TRes: Type<DeviceDetail>(),
     TBody: Type<{ location: string }>(),
   },
+  addOrganization: {
+    path: "/api/v1/facility/{facility_external_id}/device/{external_id}/add_managing_organization/",
+    method: HttpMethod.POST,
+    TRes: Type<DeviceDetail>(),
+    TBody: Type<{ managing_organization: string }>(),
+  },
```

Comment:
let's follow consistent patterns; snake case is not a convention in javascript

```suggestion
    path: "/api/v1/facility/{facilityId}/device/{id}/add_managing_organization/",
    method: HttpMethod.POST,
    TRes: Type<DeviceDetail>(),
    TBody: Type<{ managing_organization: string }>(),
  },
```

Reply:
made the changes as requested